CHANGELOG.md

```diff
@@ -1,33 +1,19 @@
 # Changelog
```

Comment:
@ThilinaManamgoda shall we please update the contributors' guide in this branch as well with the new branch name?